src/sharedData/visualization/components/VisualizationConfigForm/ColumnSelect.js

```diff
@@ -39,6 +39,7 @@ const ColumnSelect = props => {
       value={field.value}
       onChange={field.onChange}
       labelId={`${id}-label`}
+      error={!!fieldState?.error}
```

Comment:
```suggestion
      error={Boolean(fieldState?.error)}
```
Would this would the same way?

Reply:
@paulschreiber Yes, it is the same, any particular reason why you think this is better?

Reply:
Using Boolean more explicit and clearer.